README.md

```diff
@@ -138,13 +144,13 @@ After a small test, you can run the full campaign for a dedicated phase space, s
 - Dileptonic ttbar phase space : check performance for btag SFs, emu channel
 
 ```
- python runner.py --workflow ttdilep_sf --json metadata/data_Winter22_emu_BTV_Run3_2022_Comm_v1.json  --campaign Winter22Run3 --year 2022 --isJERC --isCorr  (--executor ${scaleout_site}) 
+ python runner.py --workflow ttdilep_sf --json metadata/data_Winter22_emu_BTV_Run3_2022_Comm_v1.json  --campaign Winter22Run3 --year 2022 (--executor ${scaleout_site}) 
```

Comment:
we should switch here to a Summer22 campaign as well

Reply:
Since the new sample list not ready, keep winter campaign as place holder, to be updated when files are ready

Reply:
Not sure if this is a good idea anymore, as some Winter22Run3 samples have been deleted?

Reply:
Will change to Summer22, but would like to wait until Summer22 samples are ready

Reply:
Fine by me. Once you have fixed the README typos, I will proceed with merging your PR